DANS-KNAW/easy-convert-bag-to-deposit
# DD-299: Convert titles to 'acquisitionMethod'

Fixes DD-299: Convert titles to 'acquisitionMethod'

When applied it will
--------------------
* Convert titles to 'acquisitionMethod'
* fix an oversight in #11 (a title in the profile containing more than a report number)
* 

Where should the reviewer @DANS-KNAW/easy start?
------------------------------------------------

How should this be manually tested?
-----------------------------------

See #13 

Related pull requests on github
-------------------------------
includes closed #13
might have conflicts with #12
